src/main/java/bio/overture/ego/service/UserPermissionService.java

```diff
@@ -20,13 +21,27 @@
 @Transactional
 public class UserPermissionService extends AbstractPermissionService<UserPermission> {
 
-  public UserPermissionService(UserPermissionRepository userPermissionRepository) {
-    super(UserPermission.class, userPermissionRepository);
+  private final UserPermissionRepository repository;
+
+  public UserPermissionService(UserPermissionRepository repository) {
```

Comment:
which is wierd, because `disassociatePermissionsFromUser_ExistingEntitiesAndRelationships_Success` test is passing, and i calles the  `/policies/%s/permission/users/` endpoint which calls the UserPermission service